src/main/java/mate/academy/model/User.java

```diff
@@ -0,0 +1,46 @@
+package mate.academy.model;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class User {
+    @Id
+    @Column(unique = true)
+    private String email;
```

Comment:
Lets add id field and use it as primary key(https://stackoverflow.com/questions/564310/should-i-use-a-number-or-an-email-id-to-identify-a-user-on-website)